packages/amplify-app/src/index.js

```diff
@@ -385,7 +419,15 @@ async function showAndroidHelpText() {
 }
 
 async function showIOSHelpText() {
-  // TBD
+  console.log();
+  console.log(chalk.green('Some next steps:'));
+  console.log(
+    'Setting "modelgen" to true in amplifyxc.config will allow you to generate models/entities for your GraphQL models in your next xcode build'
```

Comment:
Do we disable model generation on all the platforms?

Reply:
yes model generation needs to be called explicitly on all platforms